app/addons/documents/assets/less/documents.less

```diff
@@ -18,6 +18,7 @@
 @import "changes.less";
 @import "sidenav.less";
 @import "index-results.less";
+@import "revisionTree.less";
```

Comment:
Use hyphens instead of camelcase for LESS filenames; the queryOptions.less file in that folder is incorrect.